game_patch/hud/multi_scoreboard.cpp

```diff
@@ -65,38 +67,13 @@ int draw_scoreboard_header(int x, int y, int w, rf::NetGameType game_type, bool
 
     // Draw Game Type name
     if (!dry_run) {
-        int num_players = rf::multi_num_players();
-        std::string player_count_str =
-            " | " + std::to_string(num_players) + (num_players > 1 ? " PLAYERS" : " PLAYER");
-
-        std::string game_type_name;
-        switch (game_type) {
-            case rf::NG_TYPE_TEAMDM:
-                game_type_name = rf::strings::team_deathmatch;
-                break;
-            case rf::NG_TYPE_CTF:
-                game_type_name = rf::strings::capture_the_flag;
-                break;
-            case rf::NG_TYPE_KOTH:
-                game_type_name = "KING OF THE HILL";
-                break;
-            case rf::NG_TYPE_DC:
-                game_type_name = "DAMAGE CONTROL";
-                break;
-            case rf::NG_TYPE_REV:
-                game_type_name = "REVOLT";
-                break;
-            case rf::NG_TYPE_RUN:
-                game_type_name = "RUN";
-                break;
-            default:
-                game_type_name = rf::strings::deathmatch;
-                break;
-        }
-
-        game_type_name += player_count_str;
-
-        rf::gr::string_aligned(rf::gr::ALIGN_CENTER, x_center, cur_y, game_type_name.c_str());
+        const std::string game_info = std::format(
+            "{} \x95 {}/{} PLAYING",
+            multi_game_type_name(game_type),
```

Comment:
Could we either:
1. Move `get_game_type_string_long` to `multi.h` instead of `server.h` and use that here with `string_to_upper` instead of a new function; or
2. Since this is called per frame and it would be more performant, keep it as a separate function, but maintain a standard location and naming convention with `get_game_type_string_long` and `get_game_type_string`. Perhaps `get_game_type_string_upper` or such?